ibis/tests/expr/test_selectors.py

```diff
@@ -323,6 +324,21 @@ def test_all(penguins):
     assert penguins.select(s.all()).equals(penguins.select(penguins.columns))
 
 
+@pytest.mark.parametrize(
+    ("seq", "expected"),
+    [
+        param([0, 1, 2], (0, 1, 2), id="int_tuple"),
+        param(~s.r[[3, 4, 5]], sorted(set(range(8)) - {3, 4, 5}), id="neg_int_list"),
+        param(~s.r[3, 4, 5], sorted(set(range(8)) - {3, 4, 5}), id="neg_int_list"),
```

Comment:
```suggestion
        param(~s.r[[3, 4, 5]], sorted(set(range(8)) - {3, 4, 5}), id="neg_int_list"),
        param(~s.r[3, 4, 5], sorted(set(range(8)) - {3, 4, 5}), id="neg_tuple_list"),
```
probably not worth the additional CI run